Doc/library/random.rst

```diff
@@ -112,6 +112,15 @@ Bookkeeping functions
    :meth:`randrange` to handle arbitrarily large ranges.
 
 
+.. function:: randbytes(n)
+
+   Generate *n* random bytes.
+
+   If *n* is equal to zero, always returns an empty string for convenience.
```

Comment:
I do not think it is needed. It may confuse the user if the degenerate case be explicitly mentioned for one and only one method.

Reply:
It's kind of surprising that a function documented as "generate **random** bytes" returns a deterministic (non random) result, so I consider that it's worth it to document it.

> ... explicitly mentioned for one and only one method.

What do you mean. Currently, only randbytes() has this convenience behavior no? I asked Antoine to add a similar sentence for his getrandbits(0) PR.

Reply:
`randrange(1, 2)`, `uniform(10, 10)` and `choice([7])` all return deterministic result. I think that most if not all of random functions can return deterministic result for some arguments.

Reply:
Alright, I remove this sentence.